rotkehlchen/externalapis/coingecko.py

```diff
@@ -518,7 +518,8 @@ def query_current_price(
             self,
             from_asset: AssetWithOracles,
             to_asset: AssetWithOracles,
-    ) -> Price:
```

Comment:
you are changing the return type here and in many other function without adjusting the docstring. Adjust the docstring.